tests/L0/run_optimizers/test_lamb.py

```diff
@@ -200,16 +206,21 @@ def gen_single_type_test(self, param_type=torch.float, device="cuda"):
             ref_param, tst_param, ref_optim, tst_optim = \
                 self.gen_param_optim([tensor], lamb_option)
 
-            for i in range(self.iters):
+            if isinstance(tst_optim, apex.optimizers.FusedMixedPrecisionLamb):
+                if param_type != torch.float:
+                    # joseli: This parameter is usually passed into the constructor, 
+                    # but I do not want to change the testing interface.
+                    # As long as this parameter is set before the first call to step(), 
+                    # then it should act normally.
+                    tst_optim.reduced_precision_dtype = param_type
+            print(f"ITERS: {self.iters}")
+            for i in range(self.iters if iters is None else iters):
```

Comment:
see the comment on `test_bfloat16`

```suggestion
            for i in range(self.iters):
```